bundles/asdi/asdi-projection-change/Flyout.js

```diff
@@ -0,0 +1,32 @@
+Oskari.clazz.define('Oskari.projection.change.flyout', function (loc, options) {
+    this.loc = loc;
+    this.projectionView = null;
+    this.templates = {
+        main: jQuery('<div></div>')
+    }
+    this.element = null;
+    var me = this;
+    this.on('show', function() {
+        if (!me.getElement()) {
+            me.createUi();
+            me.setTitle(me.loc.title);
+            me.addClass(options.cls);
+            me.setContent(me.getElement());
```

Comment:
You could remove createUi() and just use: this.setContent(this.projectionView.getElement()). Or do you need the extra div-wrapper for something?